libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountIdentity.m

```diff
@@ -48,7 +48,7 @@ - (id)initWithUserId:(NSString *)userId orgId:(NSString *)orgId
 {
     self = [super init];
     if (self) {
-        _userId = userId;
+        self.userId = userId;
         _orgId = orgId;
```

Comment:
Should this be `self.orgId`?

Reply:
no override for the setter should not matter but added for consistency anyway.